configs/components/marionette-collective.rb

```diff
@@ -66,7 +66,9 @@
   when "aix"
     pkg.install_service "resources/aix/mcollective.service", nil, "mcollective"
   when "windows"
-    puts "Service files not enabled on windows"
+    # Sample - service definition for rubyw.
+    # 2nd argumenent is the Wix service specific file to be included.
+    pkg.install_service "stagedir\\ruby\\bin\\rubyw.exe", "resources/windows/msi/service.mcollective.wxs"
```

Comment:
It looks like the first entry here is hard coded into the service files. It sounds like most people want to stick with the path to nssm, ruby and rubyw hard coded, unless I'm misinterpreting things. As such, we should only be passing in the path to the service.*.wxs file here